lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -672,18 +680,20 @@ defaultTilingOfType(RewriterBase &rewriter, Operation *op,
   // Erase dialect name, such as Linalg or Tensor.
   opName.erase(0, opName.find(".") + 1);
 
-  if (tsMap.count(opName)) {
-    SmallVector<int64_t> userDefaultTileSize = tsMap.find(opName)->second;
+  if (cfg.tsMap.count(opName)) {
+    SmallVector<int64_t> userDefaultTileSize = cfg.tsMap.find(opName)->second;
     defaultTileSize =
         getAsOpFoldResult(rewriter.getI64ArrayAttr(userDefaultTileSize));
   } else {
     defaultTileSize.resize(iteratorTypes.size(), rewriter.getIndexAttr(0));
     // Try tileSize from `32` to `16`.
     SmallVector<int64_t> tsOrder = {32, 16};
-    // Only 2D tile is expected.
-    int tileDims = (isa<mlir::linalg::LinalgOp>(op) && !linalgx::isMatmulOp(op))
-                       ? cast<mlir::linalg::LinalgOp>(op).getNumReductionLoops()
-                       : 0;
+    // Record how many dims have been tiled, including fully tiled, i.e.
+    // tileSize == dimSize.
+    int nonOneTileDims =
```

Comment:
what does the new name mean?

Reply:
For a general tensor A, dimensions of which can be [A, B, C, D], would be tiled into tiled_A[E, F, G, H], where E∈[1,A], F∈[1,B],...etc. The `tileSize` controls the size of memory we use to compute or move tensors. Similarly, `nonOneTileDims` is used to represent how many valid tile dimensions(excluding `ts==1`) of `tileSize`. For example, if `tiled_A` has shape of [1,1,C,D], then the `nonOneTileDims` equals to `2`, which maybe utilized by 2D load/store instruction.